app/services/ems_infra_dashboard_service.rb

```diff
@@ -128,9 +128,11 @@ def daily_provider_metrics
     current_user = @controller.current_user
     tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone) || TimeProfile.default_time_profile
 
-    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', tp)
-                                     .where(:resource => (@ems || ManageIQ::Providers::InfraManager.all))
-                                     .where('timestamp > ?', 30.days.ago.utc).order('timestamp')
+    @daily_metrics ||= begin
+      metric_rollup_scope = MetricRollup.where(:capture_interval_name => 'daily', :time_profile => tp)
+      metric_rollup_scope = metric_rollup_scope.where(:resource => (@ems || ManageIQ::Providers::InfraManager.all))
```

Comment:
Since `@ems` is already set according to @h-kataria can we remove the `|| ManageIQ::Providers::InfraManager.all` bit here?

Reply:
This was really the source of my confusion.

Reply:
yes I will do it 👍

Reply:
@carbonin done 👍 is it ok from your point of view now ?

Reply:
Yup, looks good to me.